static/src/javascripts/projects/common/modules/experiments/tests/clever-friend-brexit.js

```diff
@@ -6,13 +6,13 @@ define([
         this.start = '2016-05-09';
         this.expiry = '2016-07-31';
         this.author = 'Anne Byrne';
-        this.description = 'Segmentation to target users with Clever Friend';
-        this.audience = 0.0;
+        this.description = 'Only expose the clever friend embed to 10% of people, by removing it for 90%';
```

Comment:
No, we're showing it to 10%. It's an interactive embedded via composer so it will be in the article by default. We have to remove it for 90% of people. A bit confusing, I agree, but seemed like the simplest way to achieve what we wanted